contracts/src/core/AlignedLayerServiceManager.sol

```diff
@@ -202,10 +204,34 @@ contract AlignedLayerServiceManager is
             senderAddress,
             batchersBalances[senderAddress]
         );
-
         payable(alignedAggregator).transfer(txCost);
     }
 
+    function isVerifierBlacklisted(
+        uint256 verifierIdx
+    ) external view validVerifierIdx(verifierIdx) returns (bool) {
+        uint256 bit = blacklistedVerifiers & (1 << verifierIdx);
+        return bit > 0;
+    }
+
+    function blacklistVerifier(
+        uint256 verifierIdx
```

Comment:
```suggestion
        uint8 verifierIdx
```

Reply:
Fixed in [ee0d97d](https://github.com/yetanotherco/aligned_layer/pull/1090/commits/ee0d97d123d0d7a144a19b35c935b4ba4305ff8a)